aten/src/ATen/core/Reduction.h

```diff
@@ -11,4 +12,5 @@ enum Reduction {
   Sum,              // Sum losses
   END
 };
-}
+} // namespace Reduction
+} // namespace at
```

Comment:
@li-roy Do you remember if there a specific reason we don't wrap `Reduction` in `at::` in the original design?

Reply:
No I don't think so.